R/utils.R

```diff
@@ -12,10 +12,26 @@
 #' multmerge(c("file1.csv", "file2.csv"))
 #' }
 multmerge = function(filenames) {
-  datalist <- lapply(filenames, function(x){read.csv(file = x, header = TRUE)})
+  datalist <- lapply(filenames, function(x){read.csv(file = x,
+                                                     header = TRUE,
+                                                     encoding = "UTF-8")})
+  if (!validate_names(datalist))
+    stop("Key translation is not the same in all files.")
   Reduce(function(x, y) {merge(x, y)}, datalist)
 }
 
+#' Validate Names
+#'
+#' Validate if n-th column of data.frames (given in list) is the same.
+#'
+#' @param list_df list of data frames.
+#' @param n inetgern denoting column name
+#'
+#' @return TRUE or FALSE
```

Comment:
this type of documentation is not particularly helpful. I usually write sth like `TRUE if names of n-th columns of data.frames is the same, FALSE otherwise.`